src/prommis/solvent_extraction/translator_leach_precip.py

```diff
@@ -0,0 +1,72 @@
+# Import Pyomo libraries
+from pyomo.environ import Set
+
+# Import IDAES cores
+from idaes.core import declare_process_block_class
+from idaes.models.unit_models.translator import TranslatorData
+
+import idaes.logger as idaeslog
+
+__author__ = "Arkoprabho Dasgupta"
+
+
+# Set up logger
+_log = idaeslog.getLogger(__name__)
+
+
+@declare_process_block_class("TranslatorLeachPrecip")
+class TranslatorDataLeachPrecip(TranslatorData):
+    """
+    Translator block representing the SX/old_leaching interface
```

Comment:
```suggestion
    Translator block representing the SX/leaching interface
```
I think this may have been left over from when I originally made the translator blocks and there were two different leaching models

Reply:
Do u want me to add this change?

Reply:
done